### DISCUSSION TITLE
issues with --time option?

### DISCUSSION THREAD

**Discussion by OMEGARAZER** [Serene-Arc/bulk-downloader-for-reddit#607] [Q&A] [answered]
Not sure if it's my use case that's causing the issue or something I'm just overlooking. I have a subreddit that I backup the posts for every 12 hours so I don't need more than 1 day worth of posts to check against but the --time option does not seem to filter down to a day. This is the command I'm running to do so.

`python -m bdfr download /Backup/Subreddit/ --time-format="%m %d %Y" --file-scheme="{TITLE} {POSTID}" --folder-scheme="{DATE}/{REDDITOR}" --no-dupes --exclude-id-file /Backup/Subreddit/downloaded.log --log=/Backup/Subreddit/log_output.txt --authenticate -v --subreddit=Subreddit --time=day`

I've tried with quotes and apostrophes around day for the time option and both seem to use the default of all as it grabs posts from about a month. I tried to capitalize day and received `Error: Invalid value for '-t' / '--time': 'DAY' is not one of 'all', 'hour', 'day', 'week', 'month', 'year'.` so it seems to be seeing it but not having it take effect. I also tried moving the --time option before --time-format and got the same results. I'm using the exclude file as well but trying to limit the amount that it needs to check in the first place.

Thanks for any advice.

**Comment by Serene-Arc** [comment, marked as answer]
The time parameter for posts does not work with the 'hot' sort filter, which is the default. You'll need to specify a different `--sort`  option, namely 'top' or 'controversial'. This was discussed in #549 but it isn't a bug, just Reddit's behaviour.

**Reply by OMEGARAZER** [reply]
Thanks, sorry I missed it already being answered.

**Reply by Serene-Arc** [reply]
All good, I need to add it to the documentation

**Comment by benmargolin** [comment]
Should --time day work with 'new'? Would seem like it should but doesn't seem to.

**Reply by Serene-Arc** [reply]
No, it doesn't. An easy way to check is to go to Reddit, click the sort button at the top, and see whether the time option shows up. It doesn't for 'new' so time is not supported for that. Our backend, praw, doesn't implement any functionality that the Reddit website doesn't.
